tokens/governance/treasury-spend.md

```diff
@@ -0,0 +1,93 @@
+    |     Treasury Address     |   [0x6d6f646C70792f74727372790000000000000000](https://moonscan.io/address/0x6d6f646C70792f74727372790000000000000000){target=_blank} |
+
+=== "Moonriver"
+    |         Variable         |                                                           Value                                                           |
+    |:------------------------:|:-------------------------------------------------------------------------------------------------------------------------:|
+    |      Proposal Bond       | {{ networks.moonriver.treasury.proposal_bond}}% of proposed spend  |
+    |     Minimum Bond     |   {{ networks.moonriver.treasury.proposal_bond_min}} MOVR |
+    |     Spend Period     |   {{ networks.moonriver.treasury.spend_period_days}} days |
+    |     Treasury Address     |   [0x6d6f646C70792f74727372790000000000000000](https://moonriver.moonscan.io/address/0x6d6f646C70792f74727372790000000000000000){target=_blank} |
+    
+=== "Moonbase Alpha"
+    |         Variable         |                                                          Value                                                          |
+    |:------------------------:|:-----------------------------------------------------------------------------------------------------------------------:|
+    |      Proposal Bond       | {{ networks.moonbase.treasury.proposal_bond}}%  |
+    |     Minimum Bond     |   {{ networks.moonbase.treasury.proposal_bond_min}} DEV |
+    |     Spend Period     |   {{ networks.moonbase.treasury.spend_period_days}} days |
+
+
+This guide will show you how to submit a proposal on Moonbase Alpha. It can be adapted for Moonbeam or Moonriver. In any case, it's recommended that you familiarize yourself with the steps of submitting a treasury proposal on Moonbase Alpha or a local dev node before taking the steps on Moonbeam or Moonriver. 
+
+## Roadmap of a Treasury Proposal {: #roadmap-of-a-treasury-proposal } 
+
+You can find a full explanation of the [happy path for a treasury proposal here](/learn/features/treasury/){target=_blank}. For more information on the process, see this [guide to the interim community treasury program](https://github.com/moonbeam-foundation/treasury/blob/main/interim/interim_treasury_proposal.md){target=_blank}.
+
+![Proposal Roadmap](/images/tokens/governance/treasury-proposals/treasury-proposal-roadmap.png)
+
+## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+
+It's highly recommended that you preface any proposal with a post on [Moonbeam's Community Forum](https://forum.moonbeam.foundation/){target=_blank}. This especially important for treasury proposals because of the irrevocable bond required to submit a treasury spend proposal. You can submit your idea for a treasury spend proposal by following [these instructions](https://moonbeam.network/blog/using-moonbeam-community-forum/){target=_blank}. 
```

Comment:
They should submit it and collect feedback for 5 days before proceeding

Reply:
update link text to use the title of the page so "the instructions in the [Using the Moonbeam Community Forum to Submit a Treasury Proposal](){} guide" instead of "[these instructions](){}"